scalyr_agent/platform_tests/run_platform_test.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2015 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------
+#
+# author: Saurabh Jain <saurabh@scalyr.com>
+
+__author__ = 'saurabh@scalyr.com'
+
+import unittest
+from subprocess import call
+import os
+
+
+class cd:
```

Comment:
`cd` is a confusing name to use because it is a common Unix command.  I would suggest you name it `WorkingDirectory`.